lib/dogtrainer/api.rb

```diff
@@ -303,7 +320,7 @@ def upsert_monitor(
 
     # Create a monitor that doesn't already exist; return its id
     #
-    # @param mon_name [String] mane of the monitor to create
+    # @param _mon_name [String] mane of the monitor to create
```

Comment:
Honestly I don't know... all I know is that when I went to build docs, I got a warning for the param name in the docstring not matching the actual param name on the method, so I fixed the docstring...